scripts/provisioning/_s3_setup

```diff
@@ -80,6 +80,11 @@ def main():
       ResetCmd(args.config).process()
       sys.stdout.write('INFO: reset successful.\n')
 
+    elif args.command == 'prepare':
```

Comment:
you have not added the prepare command to the arg parser above !!!

Reply:
fixed.